src/Tests/Shared/PlatformConditionAttribute.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[Flags]
+public enum Platform
+{
+    Windows32 = 1 << 0,
+    Windows64 = 1 << 1,
+    Windows = Windows32 | Windows64,
+    Linux32 = 1 << 2,
+    Linux64 = 1 << 3,
+    Linux = Linux32 | Linux64,
+    OSX32 = 1 << 4,
+    OSX64 = 1 << 5,
+    OSX = OSX32 | OSX64,
+    Unix32 = Linux32 | OSX32,
+    Unix64 = Linux64 | OSX64,
+    Unix = Unix32 | Unix64,
+}
+
+/// <summary>
+///     This attribute is used to execute a test class or a test
```

Comment:
Please only use a single empty space character inside a doc comment like shown in the [docs](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments).

Reply:
Yeah, I think Rider tries to implicitly adopt to some coding styles used in a project. But sometimes it just suggests things that do not match at all.

Thanks for fixing it this fast 👍